libkbfs/disk_block_cache_wrapped.go

```diff
@@ -97,14 +97,28 @@ func newDiskBlockCacheWrapped(config diskBlockCacheConfig,
 	return cache, nil
 }
 
-// DoesSyncCacheHaveSpace returns true if we have more than 1% of space left in
-// the sync cache.
-func (cache *diskBlockCacheWrapped) DoesSyncCacheHaveSpace(
-	ctx context.Context) bool {
-	if !cache.IsSyncCacheEnabled() {
-		return false
+func (cache *diskBlockCacheWrapped) getCacheLocked(
+	cacheType DiskBlockCacheType) (*DiskBlockCacheLocal, error) {
+	if cacheType == DiskBlockSyncCache {
+		if cache.syncCache == nil {
+			return nil, errors.New("Sync cache not enabled")
+		}
+		return cache.syncCache, nil
+	}
+	return cache.workingSetCache, nil
```

Comment:
Is the working set cache never nil?

Reply:
Correct.  As far as I know in production, neither is ever nil, but probably for historical reasons we still usually check the sync cache to make sure.  But I don't think there's any need to check for the working set cache -- if a `DiskBlockCache` exists at all, we can assume the working set cache is there.